sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/TransformWithStateExec.scala

```diff
@@ -26,10 +26,10 @@ import org.apache.spark.sql.catalyst.expressions.{Ascending, Attribute, Expressi
 import org.apache.spark.sql.catalyst.plans.logical._
 import org.apache.spark.sql.catalyst.plans.physical.Distribution
 import org.apache.spark.sql.execution._
+import org.apache.spark.sql.execution.streaming.StateKeyValueRowSchema.{KEY_ROW_SCHEMA, VALUE_ROW_SCHEMA}
```

Comment:
wonder whether we should clarify this a bit ? `StateKeyValueRowSchema` currently is only used with TWS right - should we rename the file/class to reflect that ?

Reply:
Yeah, makes sense. Renamed this to `TransformWithStateKeyValueRowSchema`.